src/launch/cli/github/auth/commands.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+import click
+
+from launch.github.generate_github_token import get_token
+
+
+@click.command()
+@click.option(
+    "--application_id_parameter_name",
+    required=True,
+    help=f"Name of the parameter from AWS System Manager parameter store that contains the application id of the GitHub App.",
+)
+@click.option(
+    "--installation_id_parameter_name",
+    required=True,
+    help="Name of the parameter from AWS System Manager parameter store that contains the installation id of the GitHub App.",
+)
+@click.option(
+    "--signing_cert_secret_name",
+    required=True,
+    help="Name of the parameter from AWS System Manager parameter store that contains the name of the secret from AWS Secrets Manager that has the signing certificate of the GitHub App.",
+)
+def application(
+    application_id_parameter_name: str,
+    installation_id_parameter_name: str,
+    signing_cert_secret_name: str,
+):
+    token = get_token(
```

Comment:
Consider wrapping this call in a `try`/`except` block and providing some user-friendly messaging if an exception is raised. Stack traces are hugely helpful for development purposes and finding the exact fault, but some human-focused error messaging in addition to a stack trace often goes a very long way.